deployment-templates/compute-engine/server/forseti-instance-server.py

```diff
@@ -243,8 +246,8 @@ def GenerateConfig(context):
 gsutil cp -r gs://{scanner_bucket}/rules {forseti_home}/
 
 # Download the Newest Config Validator constraints from GCS
-rm -rf {forseti_home}/policy-library
-gsutil cp -r gs://{scanner_bucket}/policy-library {forseti_home}/
+rm -rf {policy_library_home}
+gsutil cp -r gs://{scanner_bucket}/policy-library {policy_library_home}/
```

Comment:
Can you please help me understand why we are downloading the constraints from GCS?

Reply:
It was my understanding that this file only gets used by the python installer. I did not want to break the existing behavior for users today or add the policy library sync feature to the soon to be deprecated installer. Is that correct or will all this be used by terraform instances as well?

Reply:
Fair enough